modules/cudacodec/src/ffmpeg_video_source.cpp

```diff
@@ -192,14 +192,8 @@ void cv::cudacodec::detail::FFmpegVideoSource::updateFormat(const FormatInfo& vi
 
 bool cv::cudacodec::detail::FFmpegVideoSource::get(const int propertyId, double& propertyVal) const
 {
-    CV_Assert(videoCaptureParams.size() % 2 == 0);
-    for (std::size_t i = 0; i < videoCaptureParams.size(); i += 2) {
-        if (videoCaptureParams.at(i) == propertyId) {
-            propertyVal = videoCaptureParams.at(i + 1);
-            return true;
-        }
-    }
-    return false;
+    propertyVal = cap.get(propertyId);
```

Comment:
Is `VideoCapture::get` thread safe?

Reply:
There is no such guarantee in OpenCV documentation, but I have not found suspicious places in FFmpeg getProperty implementation that could produce serious issues.

Reply:
> but I have not found suspicious places in FFmpeg getProperty implementation that could produce serious issues.

I thought the same but I wasn't sure if that could change in the future if `VideoCapture::get` adds any additional properties.  Would a mutex be overkill?

Reply:
Yes. I do not see use case where it helps.